roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -144,7 +144,7 @@ docker_log_opts: "--log-opt max-size=50m --log-opt max-file=5"
 ## This string should be exactly as you wish it to appear.
 ## An obvious use case is allowing insecure-registry access
 ## to self hosted registries like so:
-docker_options: "--insecure-registry={{ kube_service_addresses }} --graph={{ docker_daemon_graph }} {{ docker_log_opts }}"
+docker_options: "--insecure-registry={{ kube_service_addresses }} --graph={{ docker_daemon_graph }} {{ docker_log_opts }} --iptables={{ docker_iptables_enabled | default('false') }}"
```

Comment:
This gets overrided in k8s-cluster.yml . You need to add it to the template `roles/docker/templates/docker-options.conf.j2`

Reply:
```roles/docker/templates/docker-options.conf.j2``` has also been changed!

Reply:
Yes. It's wrong. You need to add `--iptables={{ docker_iptables_enabled | default('false') }}` to the template, not the option. The option is overridden and this will not take affect then.